<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomDialog</class>
 <widget class="QDialog" name="CustomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="7">
    <widget class="QListWidget" name="m_listWidget">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>84</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>120</width>
       <height>90</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>General</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QStackedWidget" name="m_stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="color" stdset="0">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
     <widget class="QWidget" name="pageToDelete">
      <layout class="QGridLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="m_errorLabel">
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(180, 0, 0);
border-raduis : 4px;
padding : 7px;
background-color: rgb(222, 130, 130);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_listWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../components/xinxpluginselector.qrc"/>
  <include location="../../../../xinx/application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>label_18</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>m_stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
